### DISCUSSION TITLE
Persistent, native "overlays": is it possible?

### DISCUSSION THREAD

**Discussion by karlsander** [software-mansion/react-native-screens#1452] [Ideas]
The one feature I really like from wix/rnn is to create overlays (aka modals) as their own "native" containers that are totally independent from the stack below. So far I'm not able to accomplish that with react-native-screens, createNativeStack or the stock Modal. Some use cases for that kind of thing are:

- Notification / Toast that does not obstruct interaction, even push/poping screens
- mini player / PiP for media started on one screen that persists across navigations
- small "You don't have an internet connection" overlay
- remember Chat Heads?

I think it would be a fitting feature for react-native-screens to have, because it could make other libraries that use these kinds of patterns better.

Or am I missing something and it's already possible?

**Comment by tboba** [comment]
Hi @karlsander, sorry for the late response 😅 
As from what I can tell you're trying to create a view that will be above of all screens. First thing that comes to my mind is to just use `FullWindowOverlay` that will be on top of all views - have you tried to use that component?
